crates/fluvio-cluster/src/error.rs

```diff
@@ -18,47 +18,47 @@ use crate::runtime::local::LocalRuntimeError;
 #[derive(thiserror::Error, Debug)]
 pub enum ClusterError {
     /// An error occurred while trying to install Fluvio on Kubernetes
-    #[error("Failed to install Fluvio on Kubernetes")]
+    #[error("Failed to install Fluvio on Kubernetes: {0}")]
     InstallK8(#[from] K8InstallError),
     /// An error occurred while trying to install Fluvio locally
-    #[error("Failed to install Fluvio locally")]
+    #[error("Failed to install Fluvio locally: {0}")]
     InstallLocal(#[from] LocalInstallError),
     /// An error occurred while trying to install Fluvio system charts
-    #[error("Failed to install Fluvio system charts")]
+    #[error("Failed to install Fluvio system charts: {0}")]
     InstallSys(#[from] ChartInstallError),
     /// An error occurred while trying to uninstall Fluvio
-    #[error("Failed to uninstall Fluvio")]
+    #[error("Failed to uninstall Fluvio: {0}")]
     Uninstall(#[from] UninstallError),
-    #[error("Progress Error")]
+    #[error("Progress Error: {0}")]
     ProgressError(#[from] TemplateError),
 }
 
 /// Errors that may occur while trying to install Fluvio on Kubernetes
 #[derive(thiserror::Error, Debug)]
 pub enum K8InstallError {
     /// An error occurred with the Kubernetes config.
-    #[error("Kubernetes config error")]
+    #[error("Kubernetes config error: {0}")]
     K8ConfigError(#[from] K8ConfigError),
     /// An error occurred with the Kubernetes client.
-    #[error("Kubernetes client error")]
+    #[error("Kubernetes client error: {0}")]
     K8ClientError(#[from] K8ClientError),
     /// An error occurred while running helm.
-    #[error("Helm client error")]
+    #[error("Helm client error: {0}")]
     HelmError(#[from] HelmError),
     /// An error occurred while running helm.
-    #[error("Helm Chart error")]
+    #[error("Helm Chart error: {0}")]
     ChartError(#[from] ChartInstallError),
     /// Failed to execute a command
     #[error(transparent)]
     CommandError(#[from] CommandError),
     /// One or more pre-checks (successfully) failed when trying to start the cluster
-    #[error("Pre-checks failed during cluster startup")]
+    #[error("Pre-checks failed during cluster startup: {0:#?}")]
     FailedPrecheck(CheckStatuses),
     /// Encountered an error while performing one or more pre-checks
-    #[error("Failed to perform one or more pre-checks")]
+    #[error("Failed to perform one or more pre-checks: {0:#?}")]
```

Comment:
These kind of formats `{#?}` could span many lines causing upper logs to be missed by the Terminal buffer.

For these cases I think we could have a one liner, perhaps implementing a `Display` trait on them and have a summary of the details in a single line.

Reply:
makes sense! let me try to fix this

Reply:
Removed debug prints for now. In next iteration we can implement custom `Display`, I didn't want to do it now because it is typedef as `Vec<_>` so can't implement trait on it directly, need to make it a struct which will be a breaking change.